docs/source/examples/substrafl/go_further/run_mnist_cyclic.ipynb

```diff
@@ -254,16 +254,16 @@
         "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.                metric_functions: Optional[Union[Dict[str, Callable], List[Callable], Callable]] = None, 
</pre><p>stupid question, but given that it's just an example, could we for example just consider that we have a <code>dict</code> here? </p><p>so that the reader would be able to focus on what matters in this example? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/376/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>